cypress/components/anchor-navigation/anchor-navigation.cy.tsx

```diff
@@ -55,7 +55,7 @@ context("Testing AnchorNavigation component", () => {
     it("should pass accessibility tests for Anchor Navigation in full screen dialog", () => {
       CypressMountWithProviders(<InFullScreenDialogStory />);
 
-      getDataElementByValue("main-text")
+      return getDataElementByValue("main-text")
```

Comment:
why do we need to return this?

Reply:
I was getting a number of Typescript errors yesterday before I did an update on VS, and this solved one of them. I'm not getting them any more so I'll remove this.

Reply:
I was seeing the following error on that piece of code:
"Promise should be returned to test its fulfillment or rejection"
Adding the return fixed it.